src/Common/src/Interop/Unix/libssl/SecuritySafeHandles.cs

```diff
@@ -266,6 +276,8 @@ public override bool IsInvalid
         protected override bool ReleaseHandle()
         {
             Interop.OpenSsl.FreeSslContext(_sslContext);
+            _sslContext.DangerousRelease();
+            _sslContext.Dispose();
```

Comment:
It looks like FreeSslContext does:
```C#
Disconnect(context);
context.Dispose();
```
Shouldn't we be calling DangerousRelease on _sslContext before that?  And then why do we also call Dispose?